alchemy-web/docs/guides/cloudflare-sveltekit.md

```diff
@@ -0,0 +1,325 @@
+---
+order: 3
+title: SvelteKit
+description: Step-by-step guide to deploying a SvelteKit application to Cloudflare Workers using Alchemy with KV storage and R2 buckets.
+---
+
+# SvelteKit
+
+This guide walks through how to deploy a SvelteKit application to Cloudflare Workers with Alchemy.
+
+## Create a new SvelteKit Project
+
+Start by creating a new SvelteKit project:
+
+::: code-group
+
+```sh [bun]
+bun create svelte@latest my-sveltekit-app
+cd my-sveltekit-app
+bun install
```

Comment:
i think we're ready to merge this once we modify this to use the cloudflare getting started CLi command for SK, yeah?